Remotion/Core/ExtensibleEnums.UnitTests/TypeConversionProviderTest.cs

```diff
@@ -80,13 +80,13 @@ public void GetTypeConverter_FromString_ToExtensibleEnum ()
     [Test]
     public void GetTypeConverter_ForExtensibleEnum ()
     {
-      TypeConverter converterFirstRun = _provider.GetTypeConverter (typeof (Color));
-      TypeConverter converterSecondRun = _provider.GetTypeConverter (typeof (Color));
+      TypeConverter? converterFirstRun = _provider.GetTypeConverter (typeof (Color));

```

Comment:
Is this because of the Is.Not.Null later on? wouldn't we still want to use the NFO here?

Reply:
from ITypeConversionProvider
```csharp
    /// <returns>
    ///   A <see cref="TypeConverter"/> or <see langword="null"/> of no <see cref="TypeConverter"/> can be found.
    /// </returns>
    TypeConverter? GetTypeConverter (Type type);
```

Reply:
I was thinking TypeConverter converterFirstRun = _provider.GetTypeConverter (typeof (Color))!;

Reply:
yes, i didn't see the ! at the end. It's a shame that the nunit assertion cannot and probably will never change the nullability.